docs/monitor-kubernetes.md

```diff
@@ -0,0 +1,189 @@
+# Monitor Kubernetes 
+
+Monitoring the state of the database is crucial to timely identify and react to performance issues. [Percona Monitoring and Management (PMM) solution enables you to do just that](monitoring.md).
+
+However, the database state also depends on the state of the Kubernetes cluster itself. Hence it’s important to have metrics that can depict the state of the Kubernetes cluster.
+
+This document describes how to set up monitoring of the Kubernetes cluster health. This setup has been tested with the [PMM server](https://docs.percona.com/percona-monitoring-and-management/details/architecture.html#pmm-server) as the centralized data storage and the Victoria Metrics Operator as the metrics collector. These steps may also apply if you use another Prometheus-compatible storage.
+
+## Considerations
+
+1. In this setup we use [Victoria Metrics kubernetes monitoring stack](https://github.com/VictoriaMetrics/helm-charts/tree/master/charts/victoria-metrics-k8s-stack) Helm chart. When customizing the chart's values, consider the following:
+
+    * Since we use the PMM server for monitoring, there is no need to store the data in Victoria Metrics Operator. Therefore, set the `vmsingle.enabled` and `vmcluster.enabled` parameters in the Victoria Metrics Helm chart to `false`.
```

Comment:
```suggestion
    * Since we use the PMM server for monitoring, there is no need to store the data in Victoria Metrics Operator. Therefore, we will install Victoria Metrics Helm chart with  `vmsingle.enabled` and `vmcluster.enabled` parameters set  to `false`.
```